ansible/vars/topo_dualtor-56.yml

```diff
@@ -0,0 +1,303 @@
+    - 0.8,1.8
+    - 0.9,1.9
+    - 0.10,1.10
+    - 0.11,1.11
+    - 0.16,1.16
+    - 0.17,1.17
+    - 0.18,1.18
+    - 0.19,1.19
+    - 0.20,1.20
+    - 0.21,1.21
+    - 0.22,1.22
+    - 0.23,1.23
+    - 0.24,1.24
+    - 0.25,1.25
+    - 0.26,1.26
+    - 0.27,1.27
+    - 0.28,1.28
+    - 0.29,1.29
+    - 0.30,1.30
+    - 0.31,1.31
+    - 0.32,1.32
+    - 0.33,1.33
+    - 0.34,1.34
+    - 0.35,1.35
+    - 0.36,1.36
+    - 0.37,1.37
+    - 0.38,1.38
+    - 0.39,1.39
+    - 0.44,1.44
+    - 0.45,1.45
+    - 0.46,1.46
+    - 0.47,1.47
+    - 0.48,1.48
+    - 0.49,1.49
+    - 0.50,1.50
+    - 0.51,1.51
+    - 0.52,1.52
+    - 0.53,1.53
+    - 0.54,1.54
+    - 0.55,1.55
+  disabled_host_interfaces:
+    - 0.2,1.2
+    - 0.3,1.3
+    - 0.6,1.6
+    - 0.7,1.7
+    - 0.10,1.10
+    - 0.11,1.11
+    - 0.18,1.18
+    - 0.19,1.19
+    - 0.22,1.22
+    - 0.23,1.23
+    - 0.26,1.26
+    - 0.27,1.27
+    - 0.30,1.30
+    - 0.31,1.31
+    - 0.34,1.34
+    - 0.35,1.35
+    - 0.38,1.38
+    - 0.39,1.39
+    - 0.46,1.46
+    - 0.47,1.47
+    - 0.50,1.50
+    - 0.51,1.51
+    - 0.54,1.54
+    - 0.55,1.55
+  VMs:
+    ARISTA01T1:
+      vlans:
+        - "0.12@48"
+        - "0.13@49"
+        - "1.12@50"
+        - "1.13@51"
+      vm_offset: 0
+    ARISTA02T1:
+      vlans:
+        - "0.14@52"
+        - "0.15@53"
+        - "1.14@54"
+        - "1.15@55"
+      vm_offset: 1
+    ARISTA03T1:
+      vlans:
+        - "0.40@56"
+        - "0.41@57"
+        - "1.40@58"
+        - "1.41@59"
+      vm_offset: 2
+    ARISTA04T1:
+      vlans:
+        - "0.42@60"
+        - "0.43@61"
+        - "1.42@62"
+        - "1.43@63"
+      vm_offset: 3
+  DUT:
+    loopback:
+      ipv4:
+        - 10.1.0.32/32
+        - 10.1.0.33/32
+      ipv6:
+        - FC00:1::32/128
+        - FC00:1::33/128
+    vlan_configs:
+      default_vlan_config: one_vlan_a
+      one_vlan_a:
+        Vlan1000:
+          id: 1000
+          intfs: [0, 1, 4, 5, 8, 9, 16, 17, 20, 21, 24, 25, 28, 29, 32, 33, 36, 37, 44, 45, 48, 49, 52, 53]
+          prefix: 192.168.0.1/21
+          prefix_v6: fc02:1000::1/64
+          tag: 1000
+          mac: 00:aa:bb:cc:dd:ee
+      two_vlan_a:
+        Vlan100:
+          id: 100
+          intfs: [0, 1, 4, 5, 8, 9, 16, 17, 20, 21, 24, 25]
+          prefix: 192.168.100.1/21
+          prefix_v6: fc02:100::1/64
+          tag: 100
+        Vlan200:
+          id: 200
+          intfs: [28, 29, 32, 33, 36, 37, 44, 45, 48, 49, 52, 53]
+          prefix: 192.168.200.1/21
+          prefix_v6: fc02:200::1/64
+          tag: 200
+    tunnel_configs:
+      default_tunnel_config: tunnel_ipinip
+      tunnel_ipinip:
+        MuxTunnel0:
+          type: IPInIP
+          attach_to: Loopback0
+          dscp: uniform
+          ecn_encap: standard
+          ecn_decap: copy_from_outer
+          ttl_mode: pipe
+
+configuration_properties:
+  common:
+    dut_asn: 65100
+    dut_type: ToRRouter
+    swrole: leaf
+    nhipv4: 10.10.246.254
+    nhipv6: FC0A::FF
```

Comment:
Can you add the definitions of variables like `podset_number`, `tor_number`, etc., like this PR: https://github.com/Azure/sonic-mgmt/pull/2580

Reply:
Updated. Thanks